deis-logger-beta1-test/Chart.yaml

```diff
@@ -0,0 +1,12 @@
+name: deis-logger-beta1-test
+home: http://github.com/deis/deis
+version: 0.1.0
```

Comment:
i would rather this be v2.0.0-beta1 if we are going to use an actual version here. @sgoings @slack @mboersma opinions?